### ISSUE TITLE
Example with plugin from node_modules

### ISSUE THREAD

**Issue by codev0** [fastify/fastify-autoload#39]
Hi there,

I use predefined template generated by `fastify-cli`, and I want to add `fastify-cors` via `fastify-autoload` situated in `plugins` folder, but have no idea how to do it in right way :(.

Can you share the example how to do it?

**Comment by Eomm** [comment]
Hi,
could you add more info about what you want to do?

In general to add the `fastify-cors` you need only to run:
```
fastify generate
```

then you need only to add in the `app.js`:

```
fastify.register(require('fastify-cors'), { 
    // put your options here
    origin: "http://example.com"
  })
```

Exactly where there is the comment `// Place here your custom code!`

**Comment by codev0** [comment]
@Eomm thanks.

I've add `fastify-cors` to `app.js`, but I thought that have to add external (from npm) plugins through `fastify-autoload` which looks to plugins folder.

**Comment by Eomm** [comment]
Oh, maybe there is a misunderstanding:
the `plugin/` folder doesn't substitute the `npm install` command. In fastify we consider all a plugin (of fastify) You should read [The hitchhiker's guide to plugins](https://github.com/fastify/fastify/blob/master/docs/Plugins-Guide.md)

**Comment by codev0** [comment]
Yeah, this is misunderstanding.

Let me describe more detailed.

I've started new project using `fastify-cli`, after `fastify generate <appname>` command finish in `<appname>` folder was predefined files structure:
```
.
├── app.js
├── package.json
├── plugins
├── services
└── test
```
Then I wanted to add `fastify-cors` that provided by npm regestry, and I first thought that came to my mind was to add it to plugins folder, but I could not do it myself and so I wrote here.
